carbonmark-api/test/routes/projects/get.test.ts

```diff
@@ -293,6 +301,14 @@ describe("GET /projects", () => {
   describe("Supply filtering", () => {
     let projects: Project[];
 
+    beforeEach(() => {
+      mockICRFilters();
+    });
+
+    afterEach(() => {
+      nock.cleanAll();
+    });
```

Comment:
The build function should be doing this in `beforeEach` already

https://github.com/KlimaDAO/klimadao/blob/f973a8dc4fbe544e4d623a85a3754c6c5b912edc/carbonmark-api/test/helper.ts#L32